Server/src/main/java/net/simon987/server/assembly/instruction/IntInstruction.java

```diff
@@ -0,0 +1,34 @@
+package net.simon987.server.assembly.instruction;
+
+import net.simon987.server.assembly.CPU;
+import net.simon987.server.assembly.Instruction;
+import net.simon987.server.assembly.Status;
+/**
+ *  Software Interrupt
+ *  Pushes the flags register and the IP to the stack then
+ *   Sets the IP to 0x0200 + 2*src. 
+ *  (x2 is to align with jmp instructions)
+ *  
+ *  No reserved interrupt vectors yet
+ */
+public class IntInstruction extends Instruction{
+	
+    public static final int OPCODE = 48;
+    private CPU cpu;
+
+    public IntInstruction(CPU cpu) {
+        super("int", OPCODE);
+        this.cpu = cpu;
+    }
+	
+    @Override	
+    public Status execute(int src, Status status) {
+        cpu.Interrupt(0, src, false);
+        return status;
+    }
+
+    public Status execute(Status status) {
```

Comment:
In the Intel 8086 manual, the INT instruction has to take an operand. I think it would make sense to remove this overload.